src/components/Navbar.js

```diff
@@ -0,0 +1,15 @@
+import { Link } from 'react-router-dom';
+
+const Navbar = () => { //eslint-disable-line
+  return (
+    <div className="navBar">
+      <h1 className="logo">Bookstore CMS</h1>
+      <ul className="navItems">
+        <li className="navItem"><Link to="/Book">Book</Link></li>
+        <li className="navItem"><Link to="/calculate">Categories</Link></li>
+      </ul>
```

Comment:
- [ ] You added the wrong path for the `Categories` component, So please make the path `to="categories"` instead of `to="/calculate"` to make the app render the `Categories` component.